src/FlatbufferMessage.cpp

```diff
@@ -0,0 +1,39 @@
+#include "FlatbufferMessage.h"
+#include "FlatbufferReader.h"
+
+namespace FileWriter {
+FlatbufferMessage::FlatbufferMessage() : Valid(false) {}
+
+FlatbufferMessage::FlatbufferMessage(char const *Pointer, size_t const Size)
+    : DataPtr(const_cast<char *>(Pointer)), DataSize(Size) {
```

Comment:
Size being const has no effect?

Reply:
True, but it does not hur either. Seeing as you mentioned it, there is some other `const` weirdness on that same line. I have now fixed that.